source/portable/NetworkInterface/DriverSAM/NetworkInterface.c

```diff
@@ -685,12 +689,10 @@ static BaseType_t prvSAM_NetworkInterfaceOutput( NetworkInterface_t * pxInterfac
 static BaseType_t prvGMACInit( NetworkInterface_t * pxInterface )
 {
     NetworkEndPoint_t * pxEndPoint;
+    size_t xIndex;
```

Comment:
Well, here's what [the FreeRTOS coding standard](https://www.freertos.org/FreeRTOS-Coding-Standard-and-Style-Guide.html#Testing) says:
> Variables of type size_t are also prefixed x

Is FreeRTOS+TCP following a different standard? Is is written down somewhere? That could save us a lot of this back and forth.
I will address all of your comments on Monday when I get to my development PC. In the next few days, I will try to go over the entire PR again and fix as much of these small details as possible.

p.s As a side note, I'm starting to get pulled in different directions at work and may have to slow down my involvement with the PR. Hopefully we can button this up fairly soon.

Reply:
> Well, here's what [the FreeRTOS coding standard](https://www.freertos.org/FreeRTOS-Coding-Standard-and-Style-Guide.html#Testing) says:
> 
>> Variables of type size_t are also prefixed x

I am surprised to see that. I would have expected `ux` because `size_t` is unsigned.
`UBaseType_t` variables start with `ux`:
~~~
UBaseType_t uxIndex;
~~~

> Is FreeRTOS+TCP following a different standard? Is is written down somewhere? That could save us a lot of this back and forth.

FreeRTOS+TCP and FreeRTOS+FAT were meant to be written in the FreeRTOS style. Today there are still differences, and little by little they will disappear. What we don't like are huge PR's just to change the variable prefixes. That make it difficult to compare source versions.

Reply:
For now I think we should stick to `ux` since +TCP uses that 🙂